Description: Fix paths to UnitTest framework
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: 2017-03-30

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -6,8 +6,6 @@
     execute_process(COMMAND git submodule update --init WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}")
 endif()
 
-add_subdirectory(unittest-cpp)
-
 include_directories(
     ../include
     ./unittest-cpp
diff --git a/tests/algorithm_tests.cpp b/tests/algorithm_tests.cpp
--- a/tests/algorithm_tests.cpp
+++ b/tests/algorithm_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/gsl_algorithm>
 
 #include <array>
diff --git a/tests/assertion_tests.cpp b/tests/assertion_tests.cpp
--- a/tests/assertion_tests.cpp
+++ b/tests/assertion_tests.cpp
@@ -14,7 +14,7 @@
 // 
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h> 
+#include <unittest++/UnitTest++.h> 
 #include <gsl/gsl>
 
 using namespace gsl;
diff --git a/tests/at_tests.cpp b/tests/at_tests.cpp
--- a/tests/at_tests.cpp
+++ b/tests/at_tests.cpp
@@ -14,7 +14,7 @@
 // 
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h> 
+#include <unittest++/UnitTest++.h> 
 #include <gsl/gsl>
 #include <vector>
 #include <initializer_list>
diff --git a/tests/bounds_tests.cpp b/tests/bounds_tests.cpp
--- a/tests/bounds_tests.cpp
+++ b/tests/bounds_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/multi_span>
 #include <vector>
 
diff --git a/tests/byte_tests.cpp b/tests/byte_tests.cpp
--- a/tests/byte_tests.cpp
+++ b/tests/byte_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/gsl_byte>
 
 #include <iostream>
diff --git a/tests/multi_span_tests.cpp b/tests/multi_span_tests.cpp
--- a/tests/multi_span_tests.cpp
+++ b/tests/multi_span_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/multi_span>
 
 #include <iostream>
diff --git a/tests/notnull_tests.cpp b/tests/notnull_tests.cpp
--- a/tests/notnull_tests.cpp
+++ b/tests/notnull_tests.cpp
@@ -14,7 +14,7 @@
 // 
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h> 
+#include <unittest++/UnitTest++.h> 
 #include <gsl/gsl>
 #include <vector>
 
diff --git a/tests/owner_tests.cpp b/tests/owner_tests.cpp
--- a/tests/owner_tests.cpp
+++ b/tests/owner_tests.cpp
@@ -14,7 +14,7 @@
 // 
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h> 
+#include <unittest++/UnitTest++.h> 
 #include <gsl/gsl>
 #include <functional>
 
diff --git a/tests/span_tests.cpp b/tests/span_tests.cpp
--- a/tests/span_tests.cpp
+++ b/tests/span_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/span>
 
 #include <iostream>
diff --git a/tests/strided_span_tests.cpp b/tests/strided_span_tests.cpp
--- a/tests/strided_span_tests.cpp
+++ b/tests/strided_span_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/multi_span>
 
 #include <string>
diff --git a/tests/string_span_tests.cpp b/tests/string_span_tests.cpp
--- a/tests/string_span_tests.cpp
+++ b/tests/string_span_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <cstdlib>
 #include <gsl/string_span>
 #include <vector>
diff --git a/tests/utils_tests.cpp b/tests/utils_tests.cpp
--- a/tests/utils_tests.cpp
+++ b/tests/utils_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/gsl>
 #include <functional>
 
