Description: Fix paths to UnitTest framework
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: 2017-03-30

--- microsoft-gsl-0.1~git16a6a41.orig/tests/CMakeLists.txt
+++ microsoft-gsl-0.1~git16a6a41/tests/CMakeLists.txt
@@ -6,8 +6,6 @@ if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DI
     execute_process(COMMAND git submodule update --init WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}")
 endif()
 
-add_subdirectory(unittest-cpp)
-
 include_directories(
     ../include
     ./unittest-cpp
--- microsoft-gsl-0.1~git16a6a41.orig/tests/algorithm_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/algorithm_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/gsl_algorithm>
 
 #include <array>
--- microsoft-gsl-0.1~git16a6a41.orig/tests/assertion_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/assertion_tests.cpp
@@ -14,7 +14,7 @@
 // 
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h> 
+#include <unittest++/UnitTest++.h> 
 #include <gsl/gsl>
 
 using namespace gsl;
--- microsoft-gsl-0.1~git16a6a41.orig/tests/at_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/at_tests.cpp
@@ -14,7 +14,7 @@
 // 
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h> 
+#include <unittest++/UnitTest++.h> 
 #include <gsl/gsl>
 #include <vector>
 #include <initializer_list>
--- microsoft-gsl-0.1~git16a6a41.orig/tests/bounds_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/bounds_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/multi_span>
 #include <vector>
 
--- microsoft-gsl-0.1~git16a6a41.orig/tests/byte_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/byte_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/gsl_byte>
 
 #include <iostream>
--- microsoft-gsl-0.1~git16a6a41.orig/tests/multi_span_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/multi_span_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/multi_span>
 
 #include <iostream>
--- microsoft-gsl-0.1~git16a6a41.orig/tests/notnull_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/notnull_tests.cpp
@@ -14,7 +14,7 @@
 // 
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h> 
+#include <unittest++/UnitTest++.h> 
 #include <gsl/gsl>
 #include <vector>
 
--- microsoft-gsl-0.1~git16a6a41.orig/tests/owner_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/owner_tests.cpp
@@ -14,7 +14,7 @@
 // 
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h> 
+#include <unittest++/UnitTest++.h> 
 #include <gsl/gsl>
 #include <functional>
 
--- microsoft-gsl-0.1~git16a6a41.orig/tests/span_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/span_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/span>
 
 #include <iostream>
--- microsoft-gsl-0.1~git16a6a41.orig/tests/strided_span_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/strided_span_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/multi_span>
 
 #include <string>
--- microsoft-gsl-0.1~git16a6a41.orig/tests/string_span_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/string_span_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <cstdlib>
 #include <gsl/string_span>
 #include <vector>
--- microsoft-gsl-0.1~git16a6a41.orig/tests/utils_tests.cpp
+++ microsoft-gsl-0.1~git16a6a41/tests/utils_tests.cpp
@@ -14,7 +14,7 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include <UnitTest++/UnitTest++.h>
+#include <unittest++/UnitTest++.h>
 #include <gsl/gsl>
 #include <functional>
 
